internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -866,6 +867,127 @@ func TestAccClusterAdvancedCluster_priorityNewSchema(t *testing.T) {
 	})
 }
 
+func TestAccAdvancedCluster_oldToNewSchemaWithAutoscalingDisabled(t *testing.T) {
+	acc.SkipIfTPFAdvancedCluster(t)
+	var (
+		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName = acc.RandomProjectName()
+		clusterName = acc.RandomClusterName()
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 acc.PreCheckBasicSleep(t, nil, orgID, projectName),
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             acc.CheckDestroyCluster,
+		Steps: []resource.TestStep{
+			{
+				Config: configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, false, false),
+				Check:  checkIndependentShardScalingMode(clusterName, "CLUSTER"),
+			},
+			{
+				Config: configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, true, false),
+				Check:  checkIndependentShardScalingMode(clusterName, "CLUSTER"),
+			},
```

Comment:
I am slightly confused by the current assertion. Even if autoscaling is disabled, if a config transitions from the old sharding configuration to the new sharding configuration I would expect the auto scaling mode to change to SHARD as a PATCH is sent to the latest API. Let me know if I am missing something or interpreting the test incorrectly.

Reply:
This test case covers the case of `Old sharding configuration performing update operations: preserves symmetric auto scaling mode.` which as far as I know is the correct behavior. This same case but enabling auto-scaling will make the mode change to SHARD